### ISSUE TITLE
Dialogs in the settings screen does not follow material design used by other parts of the app

### ISSUE THREAD

**Issue by octocorvus** [GrapheneOS/Apps#203]

**Comment by thestinger** [comment]
Is this still an issue?

**Comment by r3g-5z** [comment]
@thestinger The colours match Material You, but this doesn't look like a Material list preference dialog and looks like a compat one.

![Screenshot_20230107-231857.png](https://user-images.githubusercontent.com/112147643/211180430-d4ced8d2-c6b3-4a40-9db3-6236800b6661.png)


What other dialogs look like:

![Screenshot_20230107-235158.png](https://user-images.githubusercontent.com/112147643/211181049-7f76765b-2707-4e92-b265-20677d9cfb35.png)

**Comment by r3g-5z** [comment]
Yeah the release channel one is using a Material alert dialog. https://github.com/GrapheneOS/Apps/blob/c22055401aa14b84171b2d641a6e72d0c6e26725/app/src/main/java/app/grapheneos/apps/ui/ReleaseChannelDialog.kt#L24

Not sure what the other one is using but it's definitely not the same.

**Comment by octocorvus** [comment]
Relevant discussion https://github.com/material-components/material-components-android/issues/2732

**Comment by sivansundar** [comment]
Is anybody working on this? If not, can I give it a shot?

**Comment by dot166** [comment]
i have a different solution to this problem that i use, i just took SettingsLib (that is already used for GMSCompat and other settings menus that use M3(E)) made it compile with gradle and used that in the AppStore settings menu, here is my implementation of it https://github.com/dot166/AppStore/commit/c30edad9fa77478eae5d39914ff655170abc6250, the source for the SettingsLib fork is https://github.com/dot166/platform_frameworks_base/tree/16-qpr2/packages/SettingsLib
